Networking/Networking/Remote/ShipmentsRemote.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+import Alamofire
+
+
+/// ShipmentsRemote: Remote Endpoints
+///
+public class ShipmentsRemote: Remote {
```

Comment:
Another thing that I am curious about. I have noticed that most classes (if any) are not declared as `final`. I am used to make everything final, for better encapsulation, preventing subclassing and, in the case of Swift, performance (https://developer.apple.com/swift/blog/?id=27) but I was wondering if we were avoiding final explicitly 😊

Reply:
>  I am used to make everything final, for better encapsulation, preventing subclassing and, in the case of Swift, performance

This is a great @ctarda — it's a good practice that we never adopted as a team. I would vote that we should. /cc @mindgraffiti for thoughts on this.

I'll update this PR accordingly.